internal/syscallfs/syscallfs_test.go

```diff
@@ -62,24 +64,28 @@ func testOpen_Read(t *testing.T, tmpDir string, testFS FS) {
 		requireErrno(t, syscall.ENOENT, err)
 	})
 
-	t.Run("dir exists", func(t *testing.T) {
-		f, err := testFS.OpenFile(dir, os.O_RDONLY, 0)
+	t.Run(". opens root", func(t *testing.T) {
```

Comment:
added this as even if fs.FS doesn't like ".", we don't use that abstraction internally